src/site/content/en/lighthouse-accessibility/custom-control-roles/index.md

```diff
@@ -1,11 +1,13 @@
 ---
 layout: post
```

Comment:
Wanted to mention that the Glitch example in this post is currently not working. I need to fix it.